src/main/scala/io/citrine/lolo/util/LoloPyDataLoader.scala

```diff
@@ -0,0 +1,126 @@
+package io.citrine.lolo.util
+
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
+import io.citrine.lolo.PredictionResult
+
+/**
+  * Tool used to transfer data from LoloPy to the JVM
+  *
+  * Works by receiving an byte array and converting it to Scala Sequence
+  */
+object LoloPyDataLoader {
+
+  /**
+    * Receives the feature array. Assumes it to be in a C-ordering
+    * @param input Input array as a byte array
+    * @param numAttributes Number of features in the array
+    * @param bigEndian Whether the numbers are is big-endian or not
+    * @return The array as a Scala array
+    */
+  def getFeatureArray(input: Array[Byte], numAttributes : Integer, bigEndian: Boolean) : Seq[Vector[Double]] = {
+    // Get ordering
+    val ordering = if (bigEndian) ByteOrder.BIG_ENDIAN else ByteOrder.LITTLE_ENDIAN
+
+    // Wrap the input as a buffer
+    val buffer = ByteBuffer.wrap(input).order(ordering)
+
+    // Generate digits
+    val nDigits = input.length / 8
+    (0 until nDigits).map(x => buffer.getDouble(x * 8)).grouped(numAttributes).map(_.toVector).toVector
+  }
+
+
+  /**
+    * Receive a class or weight array
```

Comment:
I don't follow what a `class array` is here.

Reply:
I'm changing it to 1D array to be more generic. What I meant by "class array" was "array holding the values of the class variable (i.e., the one to be predicted by the ML model)"